homeassistant/config.py

```diff
@@ -87,19 +87,25 @@
 """
 
 
-def _valid_customize(value):
-    """Config validator for customize."""
-    if not isinstance(value, dict):
-        raise vol.Invalid('Expected dictionary')
+CUSTOMIZE_SCHEMA_ENTRY = vol.All(
+    cv.has_at_least_one_key(CONF_ENTITY_ID),
+    cv.ordered_dict(cv.match_all, cv.string),
+    cv.validate_some_keys(
+        [CONF_ENTITY_ID], cv.list_or_comma_separated(vol.Lower)))
 
-    for key, val in value.items():
-        if not valid_entity_id(key):
-            raise vol.Invalid('Invalid entity ID: {}'.format(key))
 
-        if not isinstance(val, dict):
-            raise vol.Invalid('Value of {} is not a dictionary'.format(key))
+def _dict_to_list(inp: Any) -> List:
+    if not isinstance(inp, dict):
+        return cv.ensure_list(inp)
+    if CONF_ENTITY_ID in inp:
+        return [inp]  # sigle entry
+    res = []
 
-    return value
+    for key, val in inp.items():
```

Comment:
Not sure why it can be a number, can you maybe give an example?

are we talking about the same val as [this](https://github.com/home-assistant/home-assistant/pull/5215/files/d97d252181ec8fdc874f16dac06baeddfccbf06a#r97206037) comment?